[Question] Cannot find module `react-tooltip/dist/react-tooltip.css`

**Describe the bug**
When running tests via Jest. Get error `Cannot find module 'react-tooltip/dist/react-tooltip.css'`
As i can see, file was removed in this version.
**Version of Package**
v5.10.5

**Screenshots**
![image](https://user-images.githubusercontent.com/45854841/230649150-a853f700-3422-4463-b884-21b8dbb1a474.png)

--- a/.github/workflows/bundlesize.yaml
+++ b/.github/workflows/bundlesize.yaml
@@ -14,7 +14,7 @@ jobs:
         run: yarn install
 
       - name: Build ReactTooltip component package
-        run: yarn build-rollup
+        run: yarn build
 
       - name: Bundlesize
         run: yarn run bundlesize

--- a/package.json
+++ b/package.json
@@ -41,6 +41,7 @@
       "import": "./dist/react-tooltip.min.mjs",
       "default": "./dist/react-tooltip.min.cjs"
     },
+    "./dist/react-tooltip": "./dist/react-tooltip.min.cjs",
     "./dist/react-tooltip.css": "./dist/react-tooltip.min.css",
     "./dist/react-tooltip.d.ts": "./dist/react-tooltip.d.ts",
     "./package.json": "./package.json"

--- a/package.json
+++ b/package.json
@@ -4,9 +4,9 @@
   "description": "react tooltip component",
   "scripts": {
     "dev-rollup": "node ./prebuild.js --env=development && node --max_old_space_size=2048 ./node_modules/rollup/dist/bin/rollup -c rollup.config.dev.js --watch",
-    "build-rollup": "node ./prebuild.js --env=production && npm run types && node --max_old_space_size=2048 ./node_modules/rollup/dist/bin/rollup -c rollup.config.prod.js",
+    "build": "node ./prebuild.js --env=production && npm run types && node --max_old_space_size=2048 ./node_modules/rollup/dist/bin/rollup -c rollup.config.prod.js",
     "dev": "node ./prebuild.js --env=development && node ./esbuild.config.dev.mjs",
-    "build": "node ./prebuild.js --env=production && node ./esbuild.config.prod.mjs",
+    "build-eslint": "node ./prebuild.js --env=production && node ./esbuild.config.prod.mjs",
     "types": "node --max_old_space_size=2048 ./node_modules/rollup/dist/bin/rollup -c rollup.config.types.js",
     "eslint": "eslint --ext=js --ext=jsx --ext=ts --ext=tsx --fix ./src",
     "stylelint": "stylelint \"src/**/*.css\"",
@@ -16,7 +16,7 @@
     "esbuild": "esbuild",
     "test": "jest",
     "postbuild": "npm run types && npm run bundlesize",
-    "prepublishOnly": "npm run build-rollup"
+    "prepublishOnly": "npm run build"
   },
   "types": "dist/react-tooltip.d.ts",
   "license": "MIT",

--- a/package.json
+++ b/package.json
@@ -32,6 +32,8 @@
   "bugs": {
     "url": "https://github.com/ReactTooltip/react-tooltip/issues"
   },
+  "main": "dist/react-tooltip.min.cjs",
+  "module": "dist/react-tooltip.min.mjs",
   "exports": {
     ".": {
       "types": "./dist/react-tooltip.d.ts",

--- a/rollup.config.prod.js
+++ b/rollup.config.prod.js
@@ -48,14 +48,6 @@ const sharedPlugins = [
       'process.env.NODE_ENV': JSON.stringify('development'),
     },
   }),
-  postcss({
-    extract: 'react-tooltip.min.css', // this will generate a specific file and override on multiples build, but the css will be the same
-    autoModules: true,
-    include: '**/*.css',
-    extensions: ['.css'],
-    plugins: [],
-    minimize: true,
-  }),
   nodeResolve(),
   ts({
     typescript,
@@ -68,24 +60,47 @@ const sharedPlugins = [
     include: 'node_modules/**',
   }),
 ]
-// this step is just to build the minified css and es modules javascript
+// this step is just to build the minified javascript files
 const minifiedBuildFormats = buildFormats.map(({ file, ...rest }) => ({
   file: file.replace(/(\.[cm]?js)$/, '.min$1'),
   ...rest,
+  minify: true,
   plugins: [terser(), filesize()],
 }))
 
 const allBuildFormats = [...buildFormats, ...minifiedBuildFormats]
 
-const config = {
-  input,
-  output: allBuildFormats.map((buildFormat) => ({
-    name,
-    ...buildFormat,
-    sourcemap: true,
-  })),
-  external,
-  plugins: sharedPlugins,
-}
+const config = allBuildFormats.map(
+  ({ file, format, globals, plugins: specificPlugins, minify }) => {
+    const plugins = [
+      ...sharedPlugins,
+      postcss({
+        extract: minify ? 'react-tooltip.min.css' : 'react-tooltip.css', // this will generate a specific file and override on multiples build, but the css will be the same
+        autoModules: true,
+        include: '**/*.css',
+        extensions: ['.css'],
+        plugins: [],
+        minimize: Boolean(minify),
+      }),
+    ]
+
+    if (specificPlugins && specificPlugins.length) {
+      plugins.push(...specificPlugins)
+    }
+
+    return {
+      input,
+      output: {
+        file,
+        format,
+        name,
+        globals,
+        sourcemap: true,
+      },
+      external,
+      plugins,
+    }
+  },
+)
 
 export default config

